.dir-locals.el

```diff
@@ -2,4 +2,4 @@
 ;;; For more information see (info "(emacs) Directory Variables")
 
 ((css-mode . ((css-indent-offset . 2)))
- (html-mode . ((sgml-basic-offset . 4))))
```

Comment:
Would you mind adding this to a separate PR. I don't mind adding code for emacs, but let's leave it in separate PRs.